docs/sources/administration/correlations/correlation-configuration/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Correlation
+weight: 10
+---
+
+# Correlation
+
+Each correlation is configured with the following options:
+
+**Label**
+: Link label, shown in a visualization
```

Comment:
the visualization

Reply:
Fixed in 1f52a7e734807f8ae045f4579c01385d2b8d9b89